src/pages/home/home.ts

```diff
@@ -292,20 +290,6 @@ export class HomePage {
     });
   }
 
-  private showNewDesignSlides() {
-    if (this.appProvider.isLockModalOpen) return; // Opening a modal together with the lock modal makes the pin pad unresponsive
-    this.persistenceProvider.getNewDesignSlidesFlag().then(value => {
-      if (!value) {
-        this.persistenceProvider.setNewDesignSlidesFlag('completed');
-        const modal = this.modalCtrl.create(NewDesignTourPage, {
```

Comment:
Needs to remove `NewDesignTourPage` and `modalCtrl` from import

Reply:
done thanks